src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -166,4 +192,12 @@ private String readDocuments(Document document) {
         additionalData.put(ADDITIONAL_DATA_CASE_REFERENCE_NUMBER_KEY, claim.getReferenceNumber());
         return additionalData;
     }
+
+    private Map<String, Object> wrapGeneralLetterInMap(Claim claim) {
+        Map<String, Object> additionalData = new HashMap<>();
```

Comment:
Only the letter type is different. So all the methods can be organized to have just a type as parameter.